web/lbry-color.css

```diff
@@ -0,0 +1,153 @@
+:root {
+
+/*  General
+ *  ─────────────────────────────────── */
+
+  --lbry-white: #fff;
```

Comment:
how many of these colors are we actually using? lets just include the ones we need?

Reply:
The impression I got from the branding unification document is that files relating to branding should not be modified.